data/beaconrestapi/src/test/java/tech/pegasys/artemis/beaconrestapi/BeaconRestApiTest.java

```diff
@@ -31,57 +33,51 @@
 import tech.pegasys.artemis.storage.ChainStorageClient;
 
 class BeaconRestApiTest {
-  private final ChainStorageClient storageClient =
+  private static final ChainStorageClient storageClient =
       ChainStorageClient.memoryOnlyClient(new EventBus());
-  private final JavalinServer server = mock(JavalinServer.class);
-  private final Javalin app = mock(Javalin.class);
+  private static final JavalinServer server = mock(JavalinServer.class);
+  private static final Javalin app = mock(Javalin.class);
```

Comment:
Mocks are not safe to reuse across tests so these should not be static.

Reply:
changed to BeforeEach